### ISSUE TITLE
Remove item in cart breaks cart

### ISSUE THREAD

**Issue by mmorkt** [duncanmcclean/simple-commerce#293]
If adding some items to the cart and afterwards clicking "remove" or "clear cart" (both functions will clear the cart).
Going back to a random product and adding it to cart will course an error:

```
ErrorException
Undefined index: items  

Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
vendor/doublethreedigital/simple-commerce/src/Http/Controllers/CartItemController.php:24
```

**Comment by duncanmcclean** [comment]
Wow, that doesn't seem right. I'll have a look in a couple of hours. (sorry about all these issues 😂 )

**Comment by mmorkt** [comment]
No errors 👍  But the ability to remove a single item from the cart isn't working yet

**Comment by duncanmcclean** [comment]
Hmm, that was working yesterday for me. I'll check again in a few hours. I've re-opened this issue in the meantime.

**Comment by duncanmcclean** [comment]
Finally fixed in 38155a9
